maint/local_python/binding_common.py

```diff
@@ -72,14 +72,14 @@ def get_C_param(param, mapping):
     want_star, want_bracket = '', ''
     param_type = mapping[kind]
 
-    if param_type in G.mpix_symbols:
-        param_type = re.sub(r'MPI_', 'MPIX_', param_type)
-
     if param['func_type']:
         param_type = param['func_type']
         if mapping['_name'].startswith("BIG_"):
             param_type += "_c"
 
+    if param_type in G.mpix_symbols:
+        param_type = re.sub(r'MPI_', 'MPIX_', param_type)
+
```

Comment:
Yeah, I suspected this wasn't actually working. I wrote some tests for partitioned that worked with both MPI and MPIX prefixes, but I forgot to report it...oops.